Gems/Atom/Feature/Common/Code/Include/Atom/Feature/LightingChannel/LightingChannelConfiguration.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/base.h>
+#include <AzCore/std/containers/array.h>
+#include <AzCore/RTTI/ReflectContext.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        const uint32_t LightingChannelsCount = 5;
+
+        class LightingChannelConfiguration
+        {
+        public:
+            AZ_TYPE_INFO(LightingChannelConfiguration, "{7FFD6D01-BABE-FE35-612F-63A30925E5F7}");
+
+            AZStd::array<bool, LightingChannelsCount> m_lightingChannelFlags;
+
+            LightingChannelConfiguration();
+
+            static void Reflect(AZ::ReflectContext* context);
+
+            uint32_t GetLightingChannelMask() const { return m_mask; }
```

Comment:
This could compute the value and return a mask instead of having UpdateLightingChannelMask and m_mask. This will guarantee that everything is always synchronized without a manual update  step.